test/DynamoCoreWpfTests/ViewExtensions/WorkspaceDependencyViewExtensionTests.cs

```diff
@@ -342,5 +342,21 @@ public void VerifyExternalFileReferences()
                 Assert.Contains(dependencyInfo.Name, dependenciesList);
             }
         }
+        [Test]
+        public void GetExternalFilesShouldBailIfGraphExecuting()
+        {
+            DynamoModel.IsTestMode = false;
+
+            // Open test file to verify the external file references. 
```

Comment:
Maybe we should add a description to better understand what we are trying to test here. Not obvious from the test code